### ISSUE TITLE
JetSmartFilters AND/OR logic for checkboxes

### ISSUE THREAD

**Issue by OyarDmit** [Crocoblock/suggestions#274]
If I select multiple categories on the checkbox category filter, the choice of product is not getting more narrow. Instead, it gets bigger. My products belong to multiple categories at once, and I wish that By picking, for example, two categories on the filter, I could see products that belong to two of the categories at the same time, so other products that belong to one category but don't belong to other would not be shown

TL;DR I want to suggest adding a choice whether we want to use AND or OR logic in checkbox filter, in the dashboard of course.

**Comment by davidk82** [comment]
100% supported

**Comment by KoshaK0** [comment]
can't wait this to work... that should not be too hard to implement from my little coding experience...

**Comment by javiercriado** [comment]
AND and OR logic relations...yeah!

pointed here too:  #147

**Comment by MjHead** [comment]
Done

**Comment by vehij** [comment]
> Done
Is AND and OR logic relations between checkbox items fixed??
where can I find this option?

**Comment by Sladestyle** [comment]
I realize this is closed but...has this feature ever been implemented? I was shocked to not be able to find it anyplace within the plugin.

**Comment by Sladestyle** [comment]
> Done
> Is AND and OR logic relations between checkbox items fixed??
> where can I find this option?

Hi @vehij , did you ever figure this out? Did you find where the AND/OR logic is located? Thanks!

**Comment by EstebanToloza** [comment]
Por favor, considere añadir esta feature pronto.

**Comment by EstebanToloza** [comment]
Please consider adding this functionality soon.

**Comment by BramOTR** [comment]
Yah! Same here, really looking forward to this feature. For now please let me know if someone has an workarround.
